.github/workflows/on_pull_request_cypress.yml

```diff
@@ -82,5 +96,9 @@ jobs:
           parallel: true
           group: "Paste Actions with VRT Parallel"
           config: baseUrl=${{ steps.waitForDeployment.outputs.url }}
+          # enabled percy to run cypress for VRT
+          command-prefix: "percy exec --parallel -- npx"
         env:
-          USE_CYPRESS_EYES: true
+          USE_CYPRESS_VRT: true
+          PERCY_PARALLEL_NONCE: ${{ needs.percy-batch-id.outputs.result }}
+          PERCY_PARALLEL_TOTAL: 3
```

Comment:
question: why 3? Is bigger better here?

Reply:
Ok, so it needs to match the number of machines you parallelise on. So we split the cypress job across three machines, we need to tell Percy to expect 3 "I'm done" events so it knows when to start processing

Reply:
Good to know